docs/source/examples/layerwise-folding.md

```diff
@@ -0,0 +1,289 @@
+    ax.set_xlabel(r"$L_{G_i \theta_i}$")
+    ax.set_ylabel(r"$\eta^{(i)}$")
+    ax.set_title("Single vs. multiple folding")
+    ax.set_xticks(x, labels, rotation=60)
+
+    ax.legend()
+
+    ax.bar_label(rects1, padding=3)
+    ax.bar_label(rects2, padding=3)
+
+    fig.tight_layout()
+
+    plt.show()
+```
+
+```{code-cell} ipython3
+plot_single_vs_multiple_folding(circuit)
+```
+
+As can be seen, the amount of noise on each layer is increased if the number of
+folds on that layer are increased.
+
+## Executor
+
+Next, we define an executor function that will allow us to run our experiment
+
+```{code-cell} ipython3
+def executor(circuit: cirq.Circuit, shots: int = 10_000) -> float:
+    """Returns the expectation value to be mitigated.
+
+    Args:
+        circuit: Circuit to run.
+        shots: Number of times to execute the circuit to compute the expectation value.
+    """
+    qiskit_circuit = to_qiskit(circuit) 
+    job = qiskit.execute(
+        experiments=qiskit_circuit,
+        backend=backend,
+        noise_model=noise_model,
+        basis_gates=noise_model.basis_gates if noise_model is not None else None,
+        optimization_level=0,  # Important to preserve folded gates.
+        shots=shots,
+    )
+
+    # Convert from raw measurement counts to the expectation value
+    counts = job.result().get_counts()
+
+    expectation_value = 0.0 if counts.get("0") is None else counts.get("0") / shots
+    return expectation_value
+```
+
+## Global folding with linear extrapolation
+
+First, for comparison, we apply ZNE with global folding on the entire circuit.
+We then compare the mitigated result of applying ZNE with linear extrapolation
+against the unmitigated value.
+
+
+```{code-cell} ipython3
+unmitigated = executor(circuit)
+
+linear_factory = zne.inference.LinearFactory(scale_factors=[1.0, 1.5, 2.0, 2.5, 3.0])
+mitigated = zne.execute_with_zne(circuit, executor, factory=linear_factory)
+
+print(f"Unmitigated result {unmitigated:.3f}")
+print(f"Mitigated result {mitigated:.3f}")
+```
+
+## Layerwise folding with linear extrapolation
+
+For contrast, we apply layerwise folding on only the layer with the most noise
+and use linear extrapolation. As above, we compare the mitigated and
+unmitigated values.
+
+```{code-cell} ipython3
+# Calculate the TVDs of each layer in the circuit (with `num_folds=3`):
+tvds = tvd(circuit, num_folds=3)
+
+# Fold noisiest layer only.
+layer_to_fold = max(tvds, key=tvds.get)
+fold_layer_func = zne.scaling.get_layer_folding(layer_to_fold)
+
+mitigated = zne.execute_with_zne(circuit, executor, scale_noise=fold_layer_func, factory=linear_factory)
+print(f"Mitigated (layerwise folding) result {mitigated:.3f}")
+print(f"Unmitigated result {unmitigated:.3f}")
+```
+
+**Note**: While doing layerwise folding on the noisiest layer will, on average,
+improve the mitigated value, it still will not eclipse the benefit of doing
+global folding.
```

Comment:
Might be better to use MyST's `note` directive:
````suggestion
```{note}
While doing layerwise folding on the noisiest layer will, on average,
improve the mitigated value, it still will not eclipse the benefit of doing
global folding.
```
````